CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# CHANGELOG for ZipStream-PHP
```

Comment:
Can we try to structure the Changelog like it is proposed here?
https://keepachangelog.com/en/1.0.0/

Reply:
Did some changes to the CHANGELOG to make it look closer to the keepachangelog format. Let me know what you think.